addons/l10n_be/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+from odoo import api, fields, models, tools, _
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    company_registry = fields.Char(compute='_compute_company_registry', store='True', readonly=False)
```

Comment:
`store='True'` :boom: :fire: :fire:

Reply:
Edit: Do this modification in `account` in order to be overridden by all localisations.